pkg/controller/cloudprivateipconfig/cloudprivateipconfig_controller.go

```diff
@@ -179,24 +179,98 @@ func (c *CloudPrivateIPConfigController) SyncHandler(key string) error {
 		return err
 	}
 
-	// At most one of nodeToAdd or nodeToDel will be set
-	nodeToAdd, nodeToDel := c.computeOp(cloudPrivateIPConfig)
+	// At most one of nodeNameToAdd or nodeNameToDel will be set
```

Comment:
No longer true with your change. Please update the comments.

Reply:
I removed it and added a detailed comment to computeOp.